pages.fr/common/chown.md

```diff
@@ -11,6 +11,10 @@
 
 `chown {{utilisateur}}:{{groupe}} {{chemin/vers/fichier_ou_dossier}}`
 
+- Modifie le propriétaire et le groupe pour qu'ils aient les deux le nom `utilisateur` :
```

Comment:
I don't understand. :thinking:

PS; the English version is:
> Change the owner user and group to both have the name user:

Reply:
What about this?

> Modifie le propriétaire et le groupe pour qu'ils aient tous les deux le nom `utilisateur` :

or :

> Modifie le propriétaire et le groupe pour qu'ils aient le nom `utilisateur` :

Reply:
I think your first suggestion is clearer. Updated! Thanks!